Sources/BandcampDownloader/Windows/MainWindow.xaml.cs

```diff
@@ -977,46 +913,6 @@ public partial class MainWindow: Window
             Cursor = Cursors.Arrow;
         }
 
-        private void checkBoxResizeCoverArt_CheckedChanged(object sender, RoutedEventArgs e) {
-            if (checkBoxResizeCoverArt == null || textBoxCoverArtMaxSize == null) {
-                return;
-            }
-
-            textBoxCoverArtMaxSize.IsEnabled = checkBoxResizeCoverArt.IsChecked.Value;
-        }
-
-        private void checkBoxRetrieveFilesizes_CheckedChanged(object sender, RoutedEventArgs e) {
-            if (checkBoxRetrieveFilesizes == null) {
-                return;
-            }
-
-            UserSettings.RetrieveFilesizes = checkBoxRetrieveFilesizes.IsChecked.Value;
-        }
-
-        private void checkBoxSaveCoverArt_CheckedChanged(object sender, RoutedEventArgs e) {
-            if (checkBoxCoverArtInFolder == null || checkBoxCoverArtInTags == null) {
-                return;
-            }
-
-            if (!checkBoxCoverArtInFolder.IsChecked.Value && !checkBoxCoverArtInTags.IsChecked.Value) {
-                checkBoxConvertToJpg.IsEnabled = false;
-                checkBoxResizeCoverArt.IsEnabled = false;
-                textBoxCoverArtMaxSize.IsEnabled = false;
-            } else {
-                checkBoxConvertToJpg.IsEnabled = true;
-                checkBoxResizeCoverArt.IsEnabled = true;
-                textBoxCoverArtMaxSize.IsEnabled = true;
-            }
-        }
-
-        private void checkBoxVerboseLog_CheckedChanged(object sender, RoutedEventArgs e) {
-            this.verboseLog = checkBoxVerboseLog.IsChecked.Value;
-        }
-
-        private void checkBoxAutoScrollLog_CheckedChanged(object sender, RoutedEventArgs e) {
```

Comment:
Why is the `CheckedChanged` event removed?

Reply:
But this event isn't used to save the setting to file, but to save internally the "autoscroll log" option. "Autoscroll log" can't be disabled anymore.

Reply:
Got it. I missed that it was still using the local variable. I removed the variable and relied on the settings, so that the autoscroll/verbose options work.